integration/commands_administration_test.go

```diff
@@ -299,126 +298,186 @@ func TestCommandsAdministrationGetParameter(t *testing.T) {
-				{"size", int32(16384)},
-				{"storageSize", int32(8192)},
-				{"totalIndexSize", int32(0)},
-				{"totalSize", int32(16384)},
-				{"scaleFactor", int32(1)},
-				{"ok", float64(1)},
-			},
-		},
-		"DataSize": {
-			command: bson.D{{"dataSize", collection.Database().Name() + "." + collection.Name()}},
-			response: bson.D{
-				{"estimate", false},
-				{"size", int32(106_496)},
-				{"numObjects", int32(210)},
-				{"millis", int32(20)},
-				{"ok", float64(1)},
-			},
-		},
-		"DataSizeCollectionNotExist": {
-			command: bson.D{{"dataSize", "some-database.some-collection"}},
-			response: bson.D{
-				{"size", int32(0)},
-				{"numObjects", int32(0)},
-				{"millis", int32(20)},
-				{"ok", float64(1)},
-			},
-		},
-		"DBStats": {
-			command: bson.D{{"dbStats", int32(1)}},
-			response: bson.D{
-				{"db", collection.Database().Name()},
-				{"collections", int32(1)},
-				{"views", int32(0)},
-				{"objects", int32(43)},
-				{"avgObjSize", 481.88235294117646},
-				{"dataSize", float64(8192)},
-				{"indexes", int32(0)},
-				{"indexSize", float64(0)},
-				{"totalSize", float64(16384)},
-				{"scaleFactor", float64(1)},
-				{"ok", float64(1)},
-			},
-		},
-		"DBStatsWithScale": {
-			command: bson.D{{"dbStats", int32(1)}, {"scale", float64(1_000)}},
-			response: bson.D{
-				{"db", collection.Database().Name()},
-				{"collections", int32(1)},
-				{"views", int32(0)},
-				{"objects", int32(43)},
-				{"avgObjSize", 433.0},
-				{"dataSize", 8.192},
-				{"indexes", int32(0)},
-				{"indexSize", float64(0)},
-				{"totalSize", 16.384},
-				{"scaleFactor", float64(1_000)},
-				{"ok", float64(1)},
-			},
-		},
-		"ServerStatus": {
-			command: bson.D{{"serverStatus", int32(1)}},
-			response: bson.D{
-				{"host", ""},
-				{"version", "5.0.42"},
-				{"process", "handlers.test"},
-				{"pid", int64(0)},
-				{"uptime", int64(0)},
-				{"uptimeMillis", int64(0)},
-				{"uptimeEstimate", int64(0)},
-				{"localTime", primitive.DateTime(time.Now().Unix())},
-				{"catalogStats", bson.D{
-					{"collections", int32(1)},
-					{"capped", int32(0)},
-					{"timeseries", int32(0)},
-					{"views", int32(0)},
-					{"internalCollections", int32(0)},
-					{"internalViews", int32(0)},
-				}},
-				{"freeMonitoring", bson.D{{"state", "disabled"}}},
-				{"ok", float64(1)},
-			},
-		},
-	} {
-		name, tc := name, tc
-		t.Run(name, func(t *testing.T) {
-			t.Parallel()
+	var actual bson.D
+	command := bson.D{{"buildInfo", int32(1)}}
+	err := collection.Database().RunCommand(ctx, command).Decode(&actual)
+	require.NoError(t, err)
 
-			var actual bson.D
-			err := collection.Database().RunCommand(ctx, tc.command).Decode(&actual)
-			require.NoError(t, err)
+	doc := ConvertDocument(t, actual)
 
-			AssertEqualDocuments(t, tc.response, actual)
-		})
-	}
+	assert.Equal(t, float64(1), must.NotFail(doc.Get("ok")))
+	assert.Regexp(t, `^5\.0\.`, must.NotFail(doc.Get("version")))
+	assert.NotEmpty(t, must.NotFail(doc.Get("gitVersion")))
```

Comment:
Do cases like `gitversion` or `GITversion` need to be checked as well? Or this one is case-sensitive, and only `gitVersion` is allowed?

Reply:
> Do cases like `gitversion` or `GITversion` need to be checked as well? 

Not necessary as MongoDB has no of these fields.

> Or this one is case-sensitive, and only `gitVersion` is allowed?

It is case sensitive, the test checks that some of MongoDB status is implemented by FerretDB. 

Didn't understand your question: are you about to suggest checking whether FerretDB has no authentic document fields?

Reply:
@seeforschauer well, I remember there were some examples where both `pampam` and `pamPam` were allowed, so I wanted to check about the ones in this file. But if `gitVersion`, `sysInfo` and other mixed case fields are case-sensitive, we are safe here.

Reply:
> But if `gitVersion`, `sysInfo` and other mixed case fields are case-sensitive, we are safe here.

they are, as `types.Document` array keys and map.